src/tvpack.jl

```diff
@@ -242,7 +242,9 @@ end
 
 function bvtcdf(nu::Int, dh::Float64, dk::Float64, r::Float64)
     if nu < 1
-        return bvnuppercdf(-dh, -dk, r)
+        # The source call the commented out line below. Instead we error out
+        throw(DomainError(nu, "degrees of freedom parameter must be positive"))
+        # return bvnuppercdf(-dh, -dk, r)
```

Comment:
That is, in tvpack.f this branch is supported? And we error since `bvnuppercdf` does not exist/was not copied?

Reply:
We do have `bvnuppercdf` but I think it's weird behavior to call it when the degrees of freedom parameter is non-positive.

Reply:
I see and I agree, it seems strange to support non-positive degrees. Maybe in tvpack it is just a way to encode the limit case of infinte degrees of freedom?

In any case, shouldn't `tcdf` be changed as well for consistency? That supports non-positive degrees of freedom as well it seems.

Reply:
It should. I hadn't noticed that. Will push an update.

Reply:
Reading this again. The branch is for `nu < 1`. I'd read it as `< 0`. It doesn't makes sense at all to use the normal cdf when the degrees of freedom parameter is between zero and one. Not sure what the idea here was.